packages/terra-form-radio/tests/wdio/radio-spec.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-unused-expressions */
+/* global browser, Terra, before */
+
+describe('Radio', () => {
+  describe('Default Radio', () => {
+    before(() => browser.url('/#/tests/form-radio-tests/default'));
+    Terra.should.matchScreenshot();
+    Terra.should.beAccessible();
+    Terra.should.themeEachCustomProperty({
```

Comment:
It appears your screenshots are not updated to reflect the custom values passed in. I noticed the values you provided match the default values.

Reply:
Actually, almost all of the themeable variables are only applied on the non-default themes. For the default (open source theme) browser native styling reigns supreme. So unless we open source the other themes and inject them into the text examples, I'm not sure how we can actually test for those values.